gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
```

Comment:
Browse Gammapy code base a bit and you'll quickly see the pattern.

Here's an example of a relative import in gammapy (https://github.com/gammapy/gammapy/blob/master/gammapy/background/tests/test_models.py#L8):
```python
from ...background.models import GaussianBand2D
```

To import something from astropy in gammapy you have to use an absolute import.
Try doing a relative import for something in Astropy from gammapy ... you'll get an error saying that relative imports of things outside a given package (gammapy in this case) is not allowed.